setup.py

```diff
@@ -104,8 +104,20 @@ def configuration(parent_package='', top_path=None):
 
     return config
 
+def is_Numpy_installed():
+    try:
+        import numpy
+    except ImportError:
+        return False
+    return True
 
 def setup_package():
+
+    build_requires = []
+    is_numpy_installed = is_Numpy_installed()
+    if is_numpy_installed is False:
+        build_requires = ['numpy>=1.6.2']
```

Comment:
The point brought up is that for continuous integration that is really annoying.
Also, I think breaking will most likely result in the user doing a ``pip install numpy`` and then a ``pip install scikit-learn``, which doesn't solve anything.